examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/IConfigRefresher.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WebDemoWithEventHub
+{
+    public interface IConfigRefresher
```

Comment:
We already have `IConfigurationRefresher`. Why do we need to introduce a new interface?

Reply:
We would need an interface for DI, right? Without this, ConfigRefresher.cs class would need to implement the IConfigurationExplorer

---
In reply to: [427590458](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427590458) [](ancestors = 427590458)